website/docs/r/bigip_ltm_node.html.markdown

```diff
@@ -30,4 +35,16 @@ resource "bigip_ltm_node" "node" {
 
 * `address` - (Required) IP or hostname of the node
 
+* `connection_limit` - (Optional) Specifies the maximum number of connections allowed for the node or node address.
+
+* `dynamic_ratio` - (Optional) Specifies the fixed ratio value used for a node during ratio load balancing.
+
+* `monitor` - (Optional) specifies the name of the monitor or monitor rule that you want to associate with the node.
+
+* `rate_limit`- (Optional) Specifies the maximum number of connections per second allowed for a node or node address. The default value is 'disabled'.
+
+* `interval` - (Optional) Specifies the amount of time before sending the next DNS query. Default is 3600. This needs to be specified inside the fqdn (fully qualified domain name).
```

Comment:
I think it would be much clearer if these were indented under the `fqdn` section, otherwise it's confusing where in the hierarchy these attributes need to go.

Reply:
yes those 2 are under fqdn

Reply:
Yes, I know they are... but the above make it clear that they are.